grbllib.c

```diff
@@ -266,7 +267,7 @@ int grbl_enter (void)
 
         // Print welcome message. Indicates an initialization has occurred at power-up or with a reset.
         report_init_message();
-
+        watchdog_update();
```

Comment:
Please mention in code comment that the watchdog config used here relies on the Bootloader config for the watchdog

Reply:
okay

Reply:
done